### PR TITLE
Fix/icons

### PR DESCRIPTION
**Pull request by kevinch**
Refactor icons to respect their natural ratio.

Many earned a new prop `solid` like the _edit_ icon.

screen shot 2018-06-04 at 14 29 14

### CODE REVIEW

File: react/components/icon/README.md [hunk 1]
```diff
@@ -1,59 +1,191 @@
...
+        <div className={demoLabel}>Filter outline</div>
+        <Filter size={demoSize} />
+      </td>
+      <td>
+        <div className={demoLabel}>Filter solid</div>
+        <Filter solid size={demoSize} />
+      </td>
+      <td>
+        <div className={demoLabel}>Failure Solid</div>
+        <Failure solid size={demoSize} />
```
Comment (klzns) [inline react/components/icon/README.md]: Deixar esse Failure perto do outro Failure.

Comment (klzns) [reply react/components/icon/README.md]: Padronizar quais icones vem primeiro o Solid ou o Outline e manter o padrão.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [4.1.0] - 2018-06-04
+
+### Added
+
+* **Icon** `CarretDown`, `CarretUp`, `CarretLeft`, `CarretRigth`, `Clock`, `Cog`, `Copy`, `Delete`, `ExternalLink`, `ExternalLinkMini`, `Filter`, `Help`, `Link`, `Save`
```
Comment (leticiawanderley) [inline CHANGELOG.md]: Update snapshots too 😁

Comment (kevinch) [reply CHANGELOG.md]: Updated.

**Pull request merged**